requirements.yml

```diff
@@ -0,0 +1,11 @@
+---
+collections:
+  - name: ansible.posix
+    version: 1.4.0
+  - name: community.docker
+    version: 3.2.1
+  - name: community.general
+    version: 6.0.1
+  - name: community.rabbitmq
+    version: 1.2.3
+roles: []
```

Comment:
These are test requirements, right?
Could we move them under the `tests` dir?

Reply:
I mean, yes and no, it depends on the Ansible version (i.e., before the split from "all-in-one" to Ansible core). In the newer world of `ansible-core` modules like [`rabbitmq_plugin`](https://github.com/StackStorm/ansible-st2/blob/b9673f50d89d48bd4e1ff0a9c64cd77bb47e2e0b/roles/StackStorm.rabbitmq/tasks/main.yml#L44) have been ripped out and place in the [collection](https://docs.ansible.com/ansible/latest/collections/community/rabbitmq/rabbitmq_plugin_module.html). But yes, I think we can put `requirements.yml` anywhere in this repository.

Beyond reworking testing (with Molecule) the next big change that this repository should undergo is that it should become an actual Ansible collection.

Reply:
So sounds like `community.rabbitmq` is a core ansible-st2 requirement, how about others? 
Is the `community.docker` collection requirement for testing purposes only?

Reply:
Pushed without `community.docker`, that might have just been stray copy/paste.

Reply:
thanks, that looks good to me now as I see these are the core playbook requirements